### ISSUE TITLE
Meta: Project Governance and Structure?

### ISSUE THREAD

**Issue by dlorenc** [notaryproject/notaryproject#2]
I was a little confused as to the basic details and governance of this project. I did some looking but wasn't able to find anything definitive:

- Is this part of the OCI or the CNCF?
  - I think the answer is the CNCF based on the slack and zoom and stuff.
- Is this part of the same CNCF "project" as Notary v1?
  - The Github org name (notaryproject) implies it is the same project, but that repo is part of [TUF](https://github.com/theupdateframework/notary).
  - Who will own the proposed specifications here? The OCI or the CNCF?
- Basic governance and decision making?
  - The notary project has some stuff [here](https://github.com/theupdateframework/notary/blob/master/MAINTAINERS_RULES.md), but the TUF project has different [stuff](https://github.com/theupdateframework/tuf/blob/develop/docs/GOVERNANCE.md)
    - I think Notary is separate from TUF - they just share a GitHub org. 
  - Does this org use the Notary ones? If so - who are the MAINTAINERS?
 
Apologies if I've missed this somewhere!

**Comment by justincormack** [comment]
Yeah sorry it is a bit confusing. This is CNCF, as part of Notary project, which is indeed separate from TUF but historicallt shares an org, although this org is its one, and we may move it here. We didnt specifically appoint v2 maintainers while in the spec and prototyping phase as it seemed premature, but we do need to formalise this.

**Comment by dlorenc** [comment]
Thanks! Makes sense. I guess the only other question I have is on the proposed specs - will those be CNCF or OCI?

**Comment by SteveLasker** [comment]
We do need to formalize the process. The Notary v2 specs and implementations are part of the CNCF. To support Notary v2 in registries, we're doing the persistence, discovery, and retrieval work under OCI. They are separate in definition, but useful together. Similar to other CNCF efforts that make use of runc, distribution-spec, image-spec or artifacts.

**Comment by dlorenc** [comment]
To make sure I'm clear: the "Notary v2 Signature" and "Notary v2 Payload" formats will be part of the Notary project in the CNCF, not part of the OCI? Are these the only two specs we imagine, or will there be more?

Is the intention for these to become "Standards" or remain "specifications"? https://github.com/cncf/toc/blob/main/PRINCIPLES.md#not-a-standards-body

What are the planned "implementations" of Notary v2? The only actual code I've been able to find in here is the nv2 client: https://github.com/notaryproject/nv2. The rest of the repos are forks of other projects (some CNCF, some not), with the stated intention of upstreaming that work.

Apologies for all the basic questions here. I'm still having a really hard time understanding what the eventual "deliverables" of Notary v2 are. A year or two from now - what do people see this project as? Is there some kind of notary service operating next to registries? Will there be client-side tooling to aid in key-management and signing? What about policy for verification?

I'm not critiquing the project here, I'm really just trying to make sure everyone involved is on the same page about "what" this group is planning to eventually build, operate, and maintain.
